api/libproject/devstart.rb

```diff
@@ -946,6 +946,24 @@ def connect_to_cloud_db(cmd_name, *args)
 })
 
 
+def deploy_gcs_artifacts()
```

Comment:
Can you change this:

https://github.com/all-of-us/workbench/blob/master/api/libproject/devstart.rb#L1238

to call deploy_gcs_artifacts?

Reply:
Done